pkg/controller/pxc/controller.go

```diff
@@ -101,11 +111,11 @@ type ReconcilePerconaXtraDBCluster struct {
 	syncUsersState int32
 	serverVersion  *version.ServerVersion
 	lockers        lockStore
+	log            logr.Logger
 }
 
 func (r *ReconcilePerconaXtraDBCluster) logger(name, namespace string) logr.Logger {
-	return log.NewDelegatingLogger(log.NullLogger{}).WithName("controller_perconaxtradbcluster").
-		WithValues("cluster name", name, "namespace", namespace)
+	return log.NewDelegatingLogger(r.log).WithValues("cluster name", name, "namespace", namespace)
```

Comment:
can we add also controller everywhere (in all controllers)? like it was before
```suggestion
	return log.NewDelegatingLogger(r.log).WithValues("controller", "perconaxtradbcluster", "cluster name", name, "namespace", namespace)
```
reason: I want to understand which controller (backups or restores or the main) got some issue